CHANGELOG.rst

```diff
@@ -1,5 +1,38 @@
 Changelog
 =========
+1.7.0 (2020-03-01)
+------------------
+MLflow 1.7.0 includes several major features and improvements:
+
+Model Registry APIs: breaking changes
+- REST APIs are no longer in beta and intended to be stable until the next major version. As such, this release includes a number of breaking changes.
```

Comment:
It would be nice to mention why they changed, i.e. to clean up the API or make it more consistent with other APIs.